python/src/energuide/extracted_datatypes.py

```diff
@@ -18,11 +18,82 @@ class _Floor(typing.NamedTuple):
     length_metres: typing.Optional[float]
 
 
+class _Wall(typing.NamedTuple):
+    label: typing.Optional[str]
```

Comment:
Should this be `Optional` given it's required in `SCHEMA`?

Reply:
Something can be both `Optional` and required if we add `nullable=True` in the schema, which is something that I meant to do.

Reply:
👍

Reply:
Should we make it nullable in the schema then?